tests/cggi/boolean_line_vectorizer_lut3.mlir

```diff
@@ -0,0 +1,72 @@
+// RUN: heir-opt --cggi-boolean-line-vectorize=parallelism=4 %s | FileCheck %s
```

Comment:
could you make a small testcase that mixed lut3 with the boolean gates and makes sure that they are not vectorized together?

Reply:
Done!